ConsistencyManager/ConsistencyManager.swift

```diff
@@ -696,6 +737,28 @@ public class ConsistencyManager {
         }
     }
 
+    // MARK: Threading
+
+    /**
+     A helper function which wraps our queue.
+     You can call the closure provided by the block to check if the operation has been cancelled.
+     */
+    private func dispatchTask(task: (()->Bool)->Void) {
```

Comment:
I think the closure parameter here should have a label like `cancelled`. Also, spaces around `->` :)